AndroidSDK/src/com/leanplum/LeanplumNotificationHelper.java

```diff
@@ -25,6 +25,8 @@
 import android.content.Context;
 import android.content.res.Resources;
 import android.graphics.Bitmap;
+import android.graphics.drawable.AdaptiveIconDrawable;
```

Comment:
Won't this crash with class not found on api level <26?
If so maybe use reflection?

Reply:
There should be no problem, same as we import NotificationChannels.